packages/models-library/src/models_library/api_schemas_webserver/wallets.py

```diff
@@ -21,6 +21,10 @@ class WalletGet(OutputSchema):
     created: datetime
     modified: datetime
 
+    model_config = ConfigDict(
+        frozen=False
+    )
+
 
 class WalletGetWithAvailableCredits(WalletGet):
     available_credits: Decimal
```

Comment:
N.B. as discussed offline with @giancarloromeo pydantic 2 has a breaking change concerning how it deals with `Decimals` which will affect the behaviour of the server (https://github.com/pydantic/pydantic/issues/7120#issuecomment-2408014177). That means once this PR gets merged users will receive, say `{"my_decimal": "1.23"}` instead of `{"my_decimal": 1.23}`. This will probably affect users who use the `osparc` client.

Reply:
Checking for solution to make this back-compatible with the old behavior :ok_hand:

Reply:
This change solves our problems.

https://github.com/ITISFoundation/osparc-simcore/pull/6580/commits/ad0cf2bee81378258e5f7006f162c5134065a25d

Reply:
Very cool! Thanks a lot for the quick fix